agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -73,15 +76,29 @@ public String toString() {
     /**
      * @return a Comment as a HTML div with proper formatting to be displayed
      */
-    private String htmlFormat() {
+    private String htmlFormat(String languageCode) {
       // gets time in local time zone (default: US ET)
       LocalDateTime localTime =
           new LocalDateTime(timestamp.getTime(), DateTimeZone.forID("US/Eastern"));
       DateTimeFormatter formatter = DateTimeFormat.forPattern("h:mm a M/dd/yy");
 
       return "<div class='comment-div'>"
           + "<p class='date'>" + formatter.print(localTime) + "</p>"
-          + "<p><b>" + name + " (" + email + "):</b> <br><br>" + text + "</p></div>";
+          + "<p><b>" + name + " (" + email + "):</b></p>"
+          + "<p class='comment-text' lang=" + languageCode + ">" + text + "</p></div>";
+    }
+
+    /**
+     * Translate the text to the language of a given language code
+     */
+    public String translateText(String languageCode) {
```

Comment:
This actually does translate the text value. However, I had the String be returned as a debugging measure, so one would be able to check the result of the translated text. I'll just change the return value to void.

Reply:
Done

Reply:
Actually, I've decided to have it return text because I'm worried if I change the text every time I might get a strange telephone-like effect. I don't think this is happening, but I think it's safest if this is just a function. I'll change the documentation accordingly.

Reply:
Done